src/server/routes/api/v1/parking-lot.ts

```diff
@@ -0,0 +1,113 @@
+import httpResponse from '../../../utils/http-response';
+import type {
+  FastifyError,
+  FastifyInstance,
+  FastifyRegisterOptions,
+  FastifyReply,
+  FastifyRequest,
+} from 'fastify';
+import { UnexistentParkingLotID } from '../../../error/parking-lot.service';
+
+export default function (
+  fastify: FastifyInstance,
+  _: FastifyRegisterOptions<unknown>,
+  done: (err?: FastifyError) => void,
+): void {
+  fastify.post(
+    '/',
+    async (
+      request: FastifyRequest<{
+        Body: {
+          name: string;
+        };
+      }>,
+      reply: FastifyReply,
+    ) => {
+      try {
+        const parkingLot = await fastify.services.parkingLot.create(
+          request.body,
+        );
+
+        reply.status(201);
+        return httpResponse.created(reply, parkingLot);
```

Comment:
Using `reply.status(201)` is redundant here, as `httpResponse.created` is already setting the status to 201.
Please remove L31.

```suggestion
        return httpResponse.created(reply, parkingLot);
```